app/views/plantings/_progress.html.haml

```diff
@@ -1,10 +1,11 @@
-- if !planting.planted?
+- if planting.crop.perennial == true
```

Comment:
This can be `if planting.crop.perennial`.

Reply:
done on the other PR .